lib/utils.pm

```diff
@@ -1077,4 +1078,20 @@ sub script_run_interactive {
     }
 }
 
+
+=head2 sed_config
+  sed_config("filename",{text_to_find => text_to_replace})
+
+  generify sed usage as config file modification tool.
+  allow to modify several items in one function call 
+  by providing  text_to_find / text_to_replace as hash key/value pairs
+=cut
+sub sed_config {
```

Comment:
I would prefer a name like, `file_content_replace`, `file_replace` and `file_edit`

Reply:
+1 for `file_content_replace`  or  `file_str_replace`

Reply:
ok